apps/preview/next/pages/showcases/Filters/FiltersPanel.tsx

```diff
@@ -385,7 +385,7 @@ export default function FiltersPanel() {
                         value={value}
                         checked={price === value}
                         name="radio-price"
-                        onChange={(event) => setPrice(event.target.value)}
+                        onClick={() => (price === value ? setPrice(null) : setPrice(value))}
```

Comment:
Comment:
This way is a bit shorter (and thus more readable) imo:
```suggestion
                        onClick={() => setPrice(price === value ? null : value)}
```